### PR TITLE
Improve that basic readme shit

### PR DESCRIPTION
**Pull request by sirugh**
Get out of here with #2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,5 +15,8 @@
     "fat",
     "nasty",
     "z-index"
-  ]
+  ],
+  "dependencies": {
+    "react": "^16.9.0"
```
Comment (Nick-Gottschlich) [inline package.json]: Woah woah adding dependencies? We gotta keep this shit lightweight bro, how are we gonna capture that emerging 2g market??

Comment (sirugh) [reply package.json]: Everyone knows React is javascript, duh

**Comment by sirugh** [comment]
I can see I'm not wanted here

**Comment by Nick-Gottschlich** [comment]
I never got my white claws brah. You think open-source software is free or something??

**Comment by Nick-Gottschlich** [comment]
image

File: package.json (hunk 1 above)
Comment (longebane) [reply package.json:20]: Instead of adding it as a dependency, you should've just pasted all that react code into `utils.js`

**Pull request closed without merging**